bot/cogs/defcon.py

```diff
@@ -25,18 +26,25 @@
 BASE_CHANNEL_TOPIC = "Python Discord Defense Mechanism"
 
 
+class DefconLogInfo(Enum):
+    """Defcon Logging Information."""
```

Comment:
Since the values are just tuples, I think it is worth mentioning what each element of the tuple is for. An alternative would be to use namedtuples but perhaps that's overkill.